src/horizon.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
```

Comment:
for consistency, change to `import React from 'react'`

Reply:
fixed